modules/release-notes/ref-release-notes-new-features.adoc

```diff
@@ -3,3 +3,62 @@
 = New features
 
 This section highlights new features in {product} {product-version}.
+
+[id="enhancement-rhidp-2200"]
+== Added an individual `mountPath`
+
+This update adds an additional individual `mountPath` for extra configmaps or secrets.
+
+[id="feature-rhidp-3621"]
+== `PersistentVolumeClaims` support is available
+
+With this update, link:https://github.com/redhat-developer/rhdh-operator/blob/main/docs/configuration.md#persistentvolumeclaims[`PersistentVolumeClaims` (PVC)] support is available.
+
+[id="feature-rhidp-3817"]
+== Added Configuration Profiles
+
+With this update, there are additional link:https://github.com/redhat-developer/rhdh-operator/blob/main/docs/profiles.md[configuration profiles].
+
+[id="enhancement-rhidp-4384"]
+== Enhanced use of `kube-rbac-proxy`
+
+This update removes the `kube-rbac-proxy` sidecar container from the RHDH Operator Pod. This sidecar container protected the operator metrics endpoint. However, the main container now provides this functionality out-of-the-box. Removing this sidecar container allows for reducing the resources required to run the Operator.
+
+[id="feature-rhidp-4414"]
+== Identifying Backstage flavor for plugins by using the `developerHub.flavor` field
+
+With this update, you can use the `developerHub.flavor` field to identify whether plugins are running on {product-very-short}, RHTAP, or vanilla Backstage, as shown in the following example:
+
+.`app-config.yaml` fragment with the `developerhub.flavor` field
+
+[source,yaml]
+----
+developerHub:
+  flavor: &lt;flavor&gt;

```

Comment:
@jmagak I don't think the '<' needs to be replaced with the named character reference &lt - anyway, it's not rendering in the GitHub preview.

```suggestion
  flavor: <flavor>;

```